### PR TITLE
Merge branch features into branches

### PR DESCRIPTION
**Pull request by flov3rh**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -362,8 +362,8 @@ <h6 class="col-12 font-weight-bold"><a class="text-dark" href="#">Lorem ipsum do
 
     <!-- In the magazine -->
```
Comment (whiz25) [inline index.html]: remove this comment

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job :+1: @flov3rh 
You have done an awesome job implementing most of the requested features for the project. However, we need to fix a few things before we can merge this project.
- [x] remove all comments from your project.
- [x] look at your social media icons at ```520px``` and below. they should be like for the original site.

### Please make those changes and re-submit for another review.

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @flov3rh  🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. The navbar needs to match the one on the original website on large screens, to contain the current date and to be responsive to medium and small screens.

2. Set up Stickler to lint the CSS on your project, find out more on how to do that in [here](https://github.com/microverseinc/linters-config/tree/master/css)

3. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input)

4. Make the README file more descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference, and render the screenshots on it correctly.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello,

Thank you for the improvement made. To move on, there are some things you should change. 

- [x] Please, correct the errors from [validator.w3.org](https://validator.w3.org/nu/?doc=https%3A%2F%2Frawcdn.githack.com%2Fflov3rh%2Fusing_bootstrap%2Ff0f209cb3ff74ad6ef829d490254e44e3ab4dc54%2Findex.html)

- [x] **Navigation bar** changes with `width < 750px`. The **SIGN IN** should appear

- [x] In `width < 760px`, the images are **full-width** in section **FEATURED STORIES**

Please, make the changes and submit a new code review request.
Thanks

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap: You did it :smile: 

The project is approved :+1: 
Good luck with the next project

**Pull request merged**